sdk/attestation/Azure.Security.Attestation/src/Models/TpmAttestationResponse.cs

```diff
@@ -7,13 +7,13 @@
 namespace Azure.Security.Attestation
 {
     /// <summary>
-    /// Represents a response for a TPM attestation call. See https://docs.microsoft.com/en-us/azure/attestation/virtualization-based-security-protocol  for more information.
+    /// Represents a response for a TPM attestation call. See <seealso href="https://docs.microsoft.com/en-us/azure/attestation/virtualization-based-security-protocol"/>  for more information.
     /// </summary>
     [CodeGenModel("TpmAttestationResponse")]
-    internal partial class TpmAttestationResponse
+    public partial class TpmAttestationResponse
     {
         /// <summary>
-        /// Attestation Response data. See  https://docs.microsoft.com/en-us/azure/attestation/virtualization-based-security-protocol for more details
+        /// Attestation Response data. See  <seealso href="https://docs.microsoft.com/en-us/azure/attestation/virtualization-based-security-protocol"/> for more details.
         /// </summary>
         public ReadOnlyMemory<byte> Data { get => Base64Url.Decode(InternalData); }
```

Comment:
Should we consider caching the result of the Base64 decode or do we think it unlikely that callers will query the property multiple times?